fastify.js

```diff
@@ -639,7 +639,7 @@ function fastify (options) {
       resolveReady(fastify)
       fastify[kState].booting = false
       fastify[kState].ready = true
-      fastify[kState].promise = null
```

Comment:
was this a typo?

Reply:
I guess so. Originally it was added here: https://github.com/fastify/fastify/commit/1f24b6e787e519f1cd069fcc6f910872c0a7098d#diff-a36b53a0e851347821d855f09722921b64d2792750d1deaab126fe54964498a9